docs/unified-test-documentation/dasharo-compatibility/31L-smbios.md

```diff
@@ -0,0 +1,392 @@
+# Dasharo Compatibility: SMBIOS
+
+## Test cases
+### DMI001.001 Verify the serial number
+
+**Test description**
+
+This test aims to verify that the serial number field is filled in correctly
+according to the [Dasharo SMBIOS guidelines](https://docs.dasharo.com/dev-proc/smbios-rules/)
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+1. `OPERATING_SYSTEM` = Ubuntu 20.04
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+1. Install the `dmidecode` package: `sudo apt install dmidecode`
+1. Obtain the stock firmware dump from [cloud.3mdeb.com](https://cloud.3mdeb.com/index.php/f/391118).
+1. Extract the dump and open the `dmidecode.log` file.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Open a terminal window and execute the following command:
+
+```
+dmidecode -t system
+```
+
+1. In the `BIOS Information` section, check the `Serial number` field against the
+    [guidelines](https://docs.dasharo.com/dev-proc/smbios-rules/) and
+    `dmidecode.log`
+
+**Expected result**
+
+1. The fields should be filled in according to the Dasharo Guidelines and
```

Comment:
Consistency is questionable we use following for the same thing, which can be confusing:
- Dasharo SMBIOS guidelines
- guidelines
- Dasharo Guidelines